README.md

```diff
@@ -2,3 +2,51 @@
 
 # CHORDS MQTT Orchestrator
 
+This package provides a tool to push MQTT messages to a [CHORDS portal]().
+
+## Setup
```

Comment:
Is this setup section intended for developers or regular users?

I was thinking that for regular users, we could suggest something along these lines

```shell
python -m pip install git+https://github.com/iotwx/chords-mqtt-orchestrator.git
```

For people who are interested in modifying the source code, we would just have a dedicated section.